shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -645,16 +645,22 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   static size_t sTaskRunnerIdentifiers = 0;
   const FlutterTaskRunnerDescription cocoa_task_runner_description = {
       .struct_size = sizeof(FlutterTaskRunnerDescription),
-      .user_data = (void*)CFBridgingRetain(self),
+      .user_data = (void*)CFBridgingRetain(self),  // Keeps engine alive for post_task_callback.
```

Comment:
Let's have the comment here point to the balancing release, just as the comment below points up here.

Reply:
Done.